specification/apimanagement/resource-manager/readme.typescript.md

```diff
@@ -7,6 +7,6 @@ Please also specify `--typescript-sdks-folder=<path to root folder of your azure
 typescript:
   azure-arm: true
   package-name: "@azure/arm-apimanagement"
-  output-folder: "$(typescript-sdks-folder)/packages/@azure/arm-apimanagement"
```

Comment:
Isn't apimanagment PR https://github.com/Azure/azure-rest-api-specs/pull/5595/?

Reply:
I thought this looked familiar.  Curious why GH shows there's a delta.

Reply:
Oh because that didn't get merge yet (thought it did yesterday).  :/